app/helpers/vita_min_form_builder.rb

```diff
@@ -485,4 +517,120 @@ def warning_for_select(element_id, permitted_values, msg)
       'data-permitted': permitted_values.to_json
     )
   end
+
+  # Methods that override honeycriscp v1
+
+  alias v1_cfa_input_field cfa_input_field
+  def cfa_input_field(
+    method,
+    label_text,
+    type: "text",
+    help_text: nil,
+    options: {},
+    autofocus: nil,
+    classes: [],
+    optional: false,
+    prefix: nil
+  )
+    if prefix || options[:'data-mask']
+      return v1_cfa_input_field(
+        method,
+        label_text,
+        type: type,
+        help_text: help_text,
+        options: options,
+        autofocus: autofocus,
+        classes: classes,
+        optional: optional,
+        prefix: prefix,
+        postfix: nil, # not used in this application
+        notice: nil, # not used in this application
+      )
+    end
+
+    text_field_options = options.merge(
+      type: type,
+      autofocus: autofocus,
+      required: options[:required]
+    )
+
+    if !options[:required] && optional
+      text_field_options = text_field_options.merge(required: false)
+    end
+
+    @v2_builder.cfa_text_field(
+      method,
+      label_text,
+      help_text: help_text,
+      wrapper_options: { class: classes.join(' ') },
+      label_options: {},
+      **text_field_options
+    )
+  end
+
+  # Added help text to label and field method instead to remove it from label
+  def label_contents(label_text, help_text, optional: false, include_help_text: true)
```

Comment:
I had to include help text by default to account for a select dropdown in `app/views/hub/clients/edit_take_action.html.erb`. NJ is not using help text with either the `cfa_select` or `vita_min_state_file_select` components currently, so I did not alter these components. However, these components also need the same label/error/help text changes.